app/components/Signup.jsx

```diff
@@ -1,49 +1,96 @@
-import React from 'react'
-import { Grid, Row, Col, Form, FormGroup, FormControl, ControlLabel, Button } from 'react-bootstrap';
-
-export const Signup = ({ signup }) => (
-
-  <Form horizontal onSubmit={evt => {
-    evt.preventDefault()
-    login(evt.target.email.value, evt.target.password.value, evt.target.confirmPass)
-  } }>
-
-    <FormGroup controlId="formHorizontalEmail">
-      <Col componentClass={ControlLabel} sm={2}>E-mail Address</Col>
-      <Col sm={10}>
-        <FormControl type="email" />
-      </Col>
-    </FormGroup>
-
-    <FormGroup controlId="formHorizontalPassword">
-      <Col componentClass={ControlLabel} sm={2}>Password</Col>
-      <Col sm={10}>
-        <FormControl type="password" />
-      </Col>
-    </FormGroup>
-
-    <FormGroup controlId="formHorizontalConfirmPassword">
-      <Col componentClass={ControlLabel} sm={2}>Password</Col>
-      <Col sm={10}>
-        <FormControl type="Confirm Password" />
-      </Col>
-    </FormGroup>
-
-    <FormGroup>
-      <Col smOffset={2} sm={10}>
-        <Button type="submit">
-          Sign in
-        </Button>
-      </Col>
-    </FormGroup>
-  </Form>
-
-)
-
-import {signup} from 'APP/app/reducers/auth'
-import {connect} from 'react-redux'
-
-export default connect (
-  state => ({}),
-  {signup},
-) (Signup)
+import React from 'react';
+import { connect } from 'react-redux';
+import { addUser } from '../reducers/user-reducer';
+
+/* -----------------    COMPONENT     ------------------ */
+
+class Signup extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.onSignupSubmit = this.onSignupSubmit.bind(this);
+  }
+
+  render() {
+    const { message } = this.props;
+    return (
+      <div className="signin-container">
+        <div className="buffer local">
+          <form onSubmit={this.onSignupSubmit}>
+            <div className="form-group">
+              <label>name</label>
+              <input
+                name="name"
+                type="name"
+                className="form-control"
+                required
+              />
+            </div>
+            <div className="form-group">
+              <label>email</label>
+              <input
+                name="email"
+                type="email"
+                className="form-control"
+                required
+              />
+            </div>
+            <div className="form-group">
+              <label>password</label>
+              <input
+                name="password"
+                type="password"
+                className="form-control"
+                required
+              />
+            </div>
+            <button type="submit" className="btn btn-block btn-primary">Signup</button>
+          </form>
+        </div>
+{/*        <div className="or buffer">
```

Comment:
Keep commented out code out of master. You want to think of master as code you'd be happy for your employers to see / understand.